include/seqan/graph_align/graph_impl_align.h

```diff
@@ -666,6 +666,19 @@ getAdjacencyMatrix(Graph<Alignment<TStringSet, TCargo, TSpec> > const& g,
 
 //////////////////////////////////////////////////////////////////////////////
 
+template<typename TStringSet, typename TCargo, typename TSpec, typename TVertex, typename TVector>
+inline void
+getVertexAdjacencyVector(Graph<Alignment<TStringSet, TCargo, TSpec> > const & g,
+                         TVertex const & vertex,
+						 TVector & vectIn,
+						 TVector & vectOut)
+{
```

Comment:
Are you using tabs? This is not aligned properly...
[please change your editor so that you see all the tabs and don't commit code with tabs!]